baselayout/nsswitch.conf

```
@@ -4,7 +4,7 @@ passwd:      files usrfiles sss systemd
 shadow:      files usrfiles sss
 group:       files usrfiles sss systemd
 
-hosts:       files usrfiles dns myhostname
+hosts:       files usrfiles resolve [!UNAVAIL=return] dns myhostname
```

This is order is what Fedora uses:
```suggestion
hosts:       files usrfiles resolve [!UNAVAIL=return] myhostname dns
```
I think it makes sense to try to resolve the hostname first but anyway, by default resolved would be used and covers this.